library/rsa.c

```diff
@@ -1282,7 +1282,7 @@ int mbedtls_rsa_rsaes_oaep_encrypt(mbedtls_rsa_context *ctx,
 
 #if defined(MBEDTLS_PKCS1_V15)
 /*
- * Implementation of the PKCS#1 v2.1 RSAES-PKCS1-V1_5-ENCRYPT function
+ * Implementation of the PKCS#1 v1.5 RSAES-PKCS1-V1_5-ENCRYPT function
```

Comment:
Minor: I think the comment was correct. Version 2.1 of the standard still defines the functions from the 1.5 standard, sometimes in a more precise way. The function has `V1_5` in the name used by the standard, so it's unmabiguous.